apps/wallet-mobile/src/components/Icon/Chevron.tsx

```diff
@@ -1,16 +1,18 @@
+import {useTheme} from '@yoroi/theme'
 import * as React from 'react'
 import Svg, {Path} from 'react-native-svg'
 
 export const Chevron = ({
   direction,
-  color = 'black',
+  color,
   size = 40,
 }: {
   size?: number
-  color?: string
   backgroundColor?: string
+  color?: string
   direction: 'left' | 'right' | 'up' | 'down'
 }) => {
+  const {theme} = useTheme()
```

Comment:
maybe we should update the guidelines that consuming the Theme directly inside the Icons is allowed?

Reply:
yes! In this cases I think should be ok to have it like this